packages/bitcore-wallet-client/src/lib/api.ts

```diff
@@ -823,13 +824,14 @@ export class API extends EventEmitter {
       network,
       singleAddress: !!opts.singleAddress,
       id: opts.id,
-      usePurpose48: n > 1
+      usePurpose48: n > 1,
+      useBech32: !!opts.useBech32
     };
     this.request.post('/v2/wallets/', args, (err, res) => {
       if (err) return cb(err);
 
       var walletId = res.walletId;
-      c.addWalletInfo(walletId, walletName, m, n, copayerName);
+      c.addWalletInfo(walletId, walletName, m, n, copayerName, opts);
```

Comment:
Can we just pass  `{ useBech32: opts. useBech32 }` so it easier to follow the code?